src/commands/convert.ts

```diff
@@ -0,0 +1,56 @@
+import { promises as fPromises } from 'fs';
+import { flags } from '@oclif/command';
+import Command from '../base';
+import { ValidationError } from '../errors/validation-error';
+import { load } from '../models/SpecificationFile';
+import { SpecificationFileNotFound } from '../errors/specification-file';
+import { convert } from '@asyncapi/converter';
+
+export default class Convert extends Command {
+  static description = 'convert asyncapi documents older to newer versions';
+
+  static flags = {
+    help: flags.help({ char: 'h' }),
+    file: flags.string({ char: 'f', description: 'path to the file to convert' }),
+    output: flags.string({ char: 'o', description: 'path to the file where the result is saved' }),
+    'target-version': flags.string({ char: 't', description: 'asyncapi version to convert to', default: '2.3.0' })
```

Comment:
I think that we can have problems with hardcoded `2.3.0` version. We can forget with the 2.4.0 version release to bump this. I think that the best solution will be installing `@asyncapi/specs` package, where we have all JSON Schema for AsyncAPI spec and then make this:

```ts
import specs from '@asyncapi/specs';

const latestVersion = Object.keys(specs).pop() as string;
```

and we will be always up to date :)

Reply:
I added `declare module '@asyncapi/specs';` to the `global.d.ts` file created before and everything works fine after adding the lines you suggested as well and changing the hardcoded version for `default: latestVersion` 👌

Reply:
Awesome! :)